Added Polish translation.

FilipParyz opened the pull request:
Hi!
I've added the Polish translation to most of the sites I could think of.
I hope I've did everything all right. If there are mistakes, please help me fix them!

I want to help make QMK more accessible and translate the QMK Docs next :)

FilipParyz committed:
Added Polish translation.

FilipParyz commented:
@noroadsleft I've closed previous PR and opened this one with fresh fork of qmk.fm That should make things easier to review 👍

noroadsleft commented:
Can you go through the Polish-translated files and make sure they're all saved with UTF-8 encoding? I'm getting some weird results depending on what program I use to check these files.

Visual Studio Code is telling me this file is UTF-8 encoded, but Notepad++ says it's ISO-8859-1:

Visual Studio Code:

![image](https://user-images.githubusercontent.com/18669334/95274858-c8ae8b80-07fb-11eb-98a5-4cd4f132b89d.png)

Notepad++:

![image](https://user-images.githubusercontent.com/18669334/95274966-10351780-07fc-11eb-9bdf-2e033144e68c.png)


## Also:

You don't need to close the pull request and then open a new one when changes are requested. Just make the requested changes, save the files, then add and commit the changes in Git, and push to the same branch from which you opened the pull request.

Commit:
Changed encoding to UTF-8.

FilipParyz commented:
@noroadsleft I've checked all of them but only some needed changes pushed. I hope all of them are fixed now :)

noroadsleft commented on _i18n/pl/support.md:
```diff
@@ -0,0 +1,21 @@
+
+* Pomoc w pokryciu kosztów serwera i infrastruktury (ewentualnie dla QMK Configurator)
+* Pomoc w pokryciu kosztów prototypowania produktów marki QMK, takich jak [Proton C](https://qmk.fm/proton-c)
+* Wysyłaj produkty na spotkania, aby otrzymać prezenty/nagrody. Napisz do nas, jeśli jesteś zainteresowany! witaj pod znakiem qmk.fm
```
This is meant to be an email address.

```suggestion
* Wysyłaj produkty na spotkania, aby otrzymać prezenty/nagrody. Napisz do nas, jeśli jesteś zainteresowany! hello@qmk.fm
```

noroadsleft commented on _i18n/pl/coc.md:
```diff
@@ -0,0 +1,32 @@
+
+Celem QMK jest bycie społecznością otwartą i tolerancyjną. Zachęcamy do brania udziału każdego, niezależnie od wieku, nipełnosprawności, pochodzenia, płci fizycznej i tożsamościowej, poziomu doświadczenia, narodowości, wyglądu osobistego, przekonań politycznych, rasy, religii czy torzsamości i orientacji seksualnej.
+
+> â€śDelikatne słowo uśmierza gniew, ale ostre słowo wywołuje gniew."
```
```suggestion
> „Delikatne słowo uśmierza gniew, ale ostre słowo wywołuje gniew.”
```

noroadsleft commented on _i18n/pl/license.md:
```diff
@@ -0,0 +1,1188 @@
+
+Większość samego QMK jest objęta licencją GPL v2, ale zawiera oprogramowanie objęte różnymi innymi licencjami, z których wszystkie są krótko opisane we wnętrzu tego pakietu. Pełny tekst każdej licencji jest dostępny w repozytorium kodu i poniżej.
+
+The MIT License (MIT)
```
Would you mind creating this file again? GitHub seems to still be struggling with this file's encoding and/or contents (see the "quotes" on Lines 17, 27, etc.):

![image](https://user-images.githubusercontent.com/18669334/95720601-50dec780-0c26-11eb-8afc-2c90adedd7ba.png)

noroadsleft commented on .project:
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
This file should be removed. If you need to keep it locally, you can run `git rm --cached -- .project`.

noroadsleft commented on _i18n/pl/keyboards.md:
```diff
@@ -0,0 +1,1225 @@
+---
```
Filip, the comments on this file are largely for my own benefit. I'll leave it up to you whether to update this file further.

----

Our infrastructure doesn't appear to have any capability to provide this page in any language other than English, because the file is generated every time the `qmk/qmk_firmware` repo is updated.

noroadsleft commented on _i18n/pl/keyboards.md:
```diff
@@ -0,0 +1,1225 @@
+
+Chcesz stronę qmk.fm? [Utwórz dla niego plik `<keyboard> .md`] (https://github.com/qmk/qmk.fm/tree/gh-pages/_pages/keyboards)! (użyj innych jako przykładów / szablonów)
+
+Obecnie dostępne są tylko pliki .hex - niektóre z wymienionych tutaj mogą nie być tym, co jest potrzebne do flashowania klawiatury. W takim przypadku [proszę otworzyć problem] (https://github.com/qmk/qmk.fm/issues)!
```
This statement is from the English page, but is no longer true – `.bin` files are provided when appropriate.

noroadsleft reviewed (changes requested):
Sorry for taking so long to get back to you; I had an issue with my development environment that took a few days to sort out.

FilipParyz commented:
@noroadsleft Woah! That's a great review, thank you very much! I'm rushing to fix all the things you mentioned above. If I can I'll mention you once again when everything is done.

noroadsleft commented:
@FilipParyz Don't feel the need to rush. Just get to it when you can.

Commit:
Fixed few issues mentioned in @noroadsleft review.
 - support.md & coc.md & license.md - encoding issues left from earlier
 - .project & keyboards.md - files removed, as they're not needed

FilipParyz commented:
I think I've fixed all the issues now. If there's anything left for me to do, let me know @noroadsleft ! :)

noroadsleft commented:
Looks good. I don't speak Polish but Google seemed to think your translations were solid. :)

FilipParyz commented:
I'm Polish with more than 10 years of English classes, so I'm quite confident in my skills. Obviously I make mistakes (in writing or in encoding :D), but I'm still learning, and spending my time on mostly English-based websites, helps.

noroadsleft commented:
> I'm Polish with more than 10 years of English classes, so I'm quite confident in my skills. Obviously I make mistakes (in writing or in encoding :D), but I'm still learning, and spending my time on mostly English-based websites, helps.

Ah, that explains it. I considered that you might be native Polish, but honestly, your English was so good I couldn't tell. If you'd said you were a native English speaker I'd have probably believed you.

noroadsleft commented:
Thanks!

FilipParyz commented:
Thank you very much guys!
I'm so happy I could contribute to QMK!

PS. If there at any chance a possibility that this PR can be marked as hacktoberfest-accepted, it would be my first this year and hopefully not last. I'm trying to be more open-source friendly and what's a better place to start than with firmware I'm using for so long now :D

noroadsleft commented:
Jack added the `hacktoberfest` topic to the repository, so you should be good there.

QMK as an organization is doing Hacktoberfest differently than we did last year, and have a [QMK-specific Hacktoberfest shirt on sale through the month of October](https://olkb.com/products/2020-qmk-hacktoberfest-t-shirts). Proceeds from each shirt sold are split 50/50 between QMK and [cKeys](https://ckeys.org/). Totally optional, but I thought you might be interested.
